frontend/static/messages/pl.json

```
@@ -1453,6 +1455,11 @@
       "defaultIndicator": "Domyślne źródło wejściowe"
     },
     "main": {
+      "errors": {
```

Biome (2.4.4)

[error] The key errors was already declared.

(lint/suspicious/noDuplicateObjectKeys)